Makefile

```diff
@@ -308,12 +439,9 @@ $(addprefix .build-artefacts/annotated/, $(SRC_JS_FILES) src/TemplateCacheModule
 	${PYTHON_CMD} .build-artefacts/python-venv/bin/pip install "gspread==0.2.2"
 	touch $@
 
-.build-artefacts/ol-requirements-installation.timestamp: .build-artefacts/python-venv
```

Comment:
I have just retested without the global closure install and everything seems to work fine without the regex module.